src/app/screens/restoreFunds/restoreOrdinals/index.tsx

```diff
@@ -138,42 +137,37 @@ function RestoreOrdinals() {
     });
   };
 
-  const showContent =
-    ordinals?.length === 0 ? (
-      <>
-        <RestoreFundTitle>{t('RESTORE_ORDINAL_SCREEN.NO_FUNDS')}</RestoreFundTitle>
-        <ButtonContainer>
-          <ActionButton text={t('RESTORE_ORDINAL_SCREEN.BACK')} onPress={handleOnCancelClick} />
-        </ButtonContainer>
-      </>
-    ) : (
-      <>
-        <RestoreFundTitle>{t('RESTORE_ORDINAL_SCREEN.DESCRIPTION')}</RestoreFundTitle>
-        {ordinals?.map((ordinal) => (
-          <OrdinalRow
-            isLoading={transferringOrdinalId === ordinal.id}
-            disableTransfer={isLoading}
-            handleOrdinalTransfer={onClickTransfer}
-            ordinal={ordinal}
-            key={ordinal.id}
-          />
-        ))}
-        <ErrorContainer>
-          <ErrorText>{error}</ErrorText>
-        </ErrorContainer>
-      </>
-    );
-
   return (
     <>
       <TopRow title={t('RESTORE_ORDINAL_SCREEN.TITLE')} onClick={handleOnCancelClick} />
       <Container>
-        {!ordinals ? (
+        {ordinalsQuery.isLoading ? (
           <LoaderContainer>
             <MoonLoader color="white" size={25} />
           </LoaderContainer>
+        ) : ordinalsQuery.ordinals!.length > 0 ? (
```

Comment:
probably don't want to use the ! operator here. use ?. instead to fall through to the false case